src/main/java/swaglabs/common/CommonMethods.java

```
@@ -101,5 +103,31 @@
```

Empty catch block suppresses exception. Add appropriate exception handling or logging.